src/confirmations_impl.cc

```diff
@@ -201,15 +214,14 @@ void ConfirmationsImpl::SaveState() {
 
   std::string json = ToJSON();
   auto callback = std::bind(&ConfirmationsImpl::OnStateSaved, this, _1);
-  confirmations_client_->Save(_confirmations_name, json, callback);
+  confirmations_client_->SaveState(_confirmations_name, json, callback);
 
-  UpdateConfirmationsIsReadyStatus();
+  NotifyAdsIfConfirmationsIsReady();
```

Comment:
naming convention for this would be `MaybeNotifyConfirmationsIsReady`

Reply:
`MaybeNotifyConfirmationsIsReady` does not seem right to me either, what about `UpdateConfirmationsStatus`